langchain4j-bedrock/src/test/java/dev/langchain4j/model/bedrock/BedrockStreamingChatModelIT.java

```diff
@@ -0,0 +1,57 @@
+package dev.langchain4j.model.bedrock;
+
+import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.model.StreamingResponseHandler;
+import dev.langchain4j.model.output.Response;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import software.amazon.awssdk.regions.Region;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class BedrockStreamingChatModelIT {
+    @Test
+    @Disabled("To run this test, you must have provide your own access key, secret, region")
+    void testBedrockAnthropicStreamingChatModel() throws ExecutionException, InterruptedException, TimeoutException {
+        BedrockAnthropicStreamingChatModel bedrockChatModel = BedrockAnthropicStreamingChatModel
+                .builder()
+                .temperature(0.50f)
+                .maxTokens(300)
+                .region(Region.US_EAST_1)
+                .maxRetries(1)
+                .build();
+
+        CompletableFuture<String> futureAnswer = new CompletableFuture<>();
```

Comment:
minor: please use `TestStreamingResponseHandler`, see `MistralAiStreamingChatModelIT` as example.
you will need to import
```
        <dependency>
            <groupId>dev.langchain4j</groupId>
            <artifactId>langchain4j-core</artifactId>
            <classifier>tests</classifier>
            <type>test-jar</type>
            <scope>test</scope>
        </dependency>
```